docs/sources/features/datasources/opentsdb.md

```diff
@@ -50,7 +50,7 @@ The autocomplete only works if the OpenTSDB suggest api is enabled.
 ## Templating queries
 
 Instead of hard-coding things like server, application and sensor name in you metric queries you can use variables in their place.
```

Comment:
```suggestion
Instead of hard-coding things like server, application and sensor name in your metric queries you can use variables in their place.
```